Classes/Tcemain.php

```diff
@@ -379,13 +383,13 @@ protected function shouldFixCaches($tableName, array $databaseData)
     protected function markCachesDirty($tableName, $recordId, &$reference)
     {
         if ($tableName == 'pages') {
-            $cache = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('tx_realurl_cachemgmt', $GLOBALS ['BE_USER']->workspace, 0);
+            $cache = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(Cachemgmt::class, $GLOBALS ['BE_USER']->workspace, 0);
```

Comment:
Add use statement instead of fully qualified namespace?

Reply:
Done